config/webpack.config.js

```diff
@@ -194,14 +194,19 @@ module.exports = env => {
         /react-intl\/locale-data/,
         supportedLanguagesRE
       ),
+      new webpack.optimize.CommonsChunkPlugin({
+        name: 'vendor',
+        minChunks: Infinity,
+      }),
+      new webpack.optimize.CommonsChunkPlugin({
+        name: 'foreman-react',
+        chunks: ['bundle', ...Object.keys(pluginEntries)],
+        minChunks: (module) =>
+        module.resource && (/(foremanReact|react_app)/).test(module.resource),
```

Comment:
I'm worried this might cause issues during build for plugins, when optimizing the chunks differently - similar to what we saw for vendor.js in the past.

Reply:
I'm researching around, thanks